content/applications/inventory_and_mrp/purchase/advanced/procurement_expenses_report.rst

```diff
@@ -0,0 +1,113 @@
+.. |caret| replace:: :icon:`fa-caret-down` :guilabel:`(down)` icon
+.. |per| replace:: procurement expenses report
+
+With the *Purchase* application, users can monitor procurement expenses over time. This report helps
+companies track and analyze spending, identify cost-saving opportunities, and ensure efficient
+budget management.
+
+Create procurement expenses report
+==================================
+
+To create a vendor costs report, first navigate to :menuselection:`Purchase app --> Reporting -->
+Purchase` to open the :guilabel:`Purchase Analysis` dashboard. By default, the dashboard displays a
+line chart overview of the :guilabel:`Untaxed Total` of POs (Purchase Orders) with a
+:guilabel:`Confirmation Date` for the current month, or of RFQs (Requests for Quotation) with a
+status of *Draft*, *Sent*, or *Cancelled*.
+
+Add filters and groups
+----------------------
+
+On the top-right, click the :icon:`oi-view-pivot` :guilabel:`(pivot)` icon to switch to pivot view.
+
+Remove any default filters from the :guilabel:`Search...` bar. Then, click the |caret| to open the
+drop-down menu that contains the :guilabel:`Filters`, :guilabel:`Group By`, and
+:guilabel:`Favorites` columns.
+
+.. note::
+   Unless otherwise specified, the report displays data from both |RFQs| and |POs|. This can be
+   changed by selecting either :guilabel:`Requests for Quotation` or :guilabel:`Purchase Orders`
+   under the :guilabel:`Filters` column.
+
+Under the :guilabel:`Filters` column, select a date range to use for comparison. The report can be
+filtered by either :guilabel:`Order Date` or :guilabel:`Confirmation Date`. Choose one from the
+list, and click the |caret| to specify the date range, either by month, quarter, or year.
+
+Next, under the :guilabel:`Group by` column, select :guilabel:`Vendor`. Then, select
+:guilabel:`Product Category`, which is also located in the :guilabel:`Group By` column.
+
+.. note::
+   The selections under the :guilabel:`Group By` heading can be altered depending on the needs of
+   the individual company. For example, selecting :guilabel:`Product` instead of :guilabel:`Product
+   Category` provides a more in depth look at the performance of specific items, instead of an
+   entire category.
+
+Next, make a selection under the :guilabel:`Comparison` heading. These options are only available
+after the date range is selected under the :guilabel:`Filters` column, and vary based on that range.
+:guilabel:`Previous Period` adds a comparison to the previous period, such as the last month or
+quarter. :guilabel:`Previous Year` compares the same time period from the previous year.
+
+.. note::
+   While multiple time-based filters can be added at once, only one comparison can be selected at a
+   time.
+
+.. image:: procurement_expenses_report/filters-groups.png
+   :align: center
+   :alt: The drop-down menu of filters, group by and comparison options for the procurement expenses
+         report.
+
+Add measures
```

Comment:
Oh wow! I hadn’t really explored this feature in my own reporting docs before. It's a great learning experience and will definitely enhance my docs :blush: